pages/workspace/workspace.html.js

```diff
@@ -16,7 +16,8 @@ function Workspace({ navTitle, showLogs, resLocals }) {
       <body
         class="d-flex flex-column h-100"
         data-workspace-id="${resLocals.workspace_id}"
-        data-heartbeat-interval="${resLocals.workspaceHeartbeatIntervalSec}"
+        data-heartbeat-interval="${heartbeatIntervalSec}"
+        data-visibility-timeout="${visibilityTimeoutSec}"
```

Comment:
When possible I'd like to carry the units along to make later code easier to read (do I need a `*1000` or not?).
```suggestion
        data-heartbeat-interval-sec="${heartbeatIntervalSec}"
        data-visibility-timeout-sec="${visibilityTimeoutSec}"
```

Reply:
Done in c8b640d.